solr/test-framework/src/java/org/apache/solr/schema/SortableBinaryField.java

```diff
@@ -70,12 +70,14 @@ public BinarySortField(final String field, final boolean reverse) {
           field,
           new FieldComparatorSource() {
             @Override
-            public FieldComparator.TermOrdValComparator newComparator(
+            public TermOrdValComparator newComparator(
                 final String fieldname,
                 final int numHits,
                 final boolean enableSkipping,
                 final boolean reversed) {
-              return new FieldComparator.TermOrdValComparator(numHits, fieldname);
+              final boolean sortMissingLast = true; // TODO
```

Comment:
What to do (pun intended) about this?

Reply:
Looks like `false` is the existing behaviour: https://github.com/apache/lucene/blob/releases/lucene/9.3.0/lucene/core/src/java/org/apache/lucene/search/FieldComparator.java#L284
```suggestion
              final boolean sortMissingLast = false;
```